flink/v1.20/flink-runtime/NOTICE

```diff
@@ -7,10 +6,48 @@ The Apache Software Foundation (http://www.apache.org/).
 
 --------------------------------------------------------------------------------
 
-This binary artifact includes Apache ORC with the following in its NOTICE file:
+This binary artifact includes Apache Avro 1.7.4 with the following in its NOTICE file:
```

Comment:
Hey should upgrade :)

Reply:
it's actually overrided by 1.12.0. I'm fixing.